lblod/frontend-embeddable-notule-editor
# Add dev docker-compose config, with lpdc service

## Overview
Add an lpdc proxy running ~~on a custom port~~ a path, using nginx-proxy settings (we don't seem to have wildcard DNS domains, so can't use a subdomain without setting up a new record).

edit: The change to using a subroute means that we can now actually have https on it, as well as not publishing weird ports.

~~Draft until the api-proxy change is released.~~

##### connected issues and PRs:
Needs an 'api-proxy' image with support for setting allowed origin header: https://github.com/lblod/api-proxy-service/pull/3

### Setup
Locally should just work, as the change has been deployed and it will use this api. To test fully locally you'll need to add a compose.override with a valid key for lpdc, publish a port for lpdc-service and to modify the config of the lpdc plugin to point to this localhost port. ~~Since the api-proxy change hasn't been merged, you'll need to add an override pointing it to a local copy of the repo with the change:~~

### How to test/reproduce
Open the test app and go to either of the plugins pages or the multiple growing editors page and verify that `insert lpdc` loads the test data.

### Challenges/uncertainties
N/A

### Checks PR readiness
